deltachat-ios/en.lproj/Localizable.strings

```diff
@@ -240,8 +240,7 @@
 // search
 "search" = "Search";
 "search_explain" = "Search for chats, contacts, and messages";
-"search_no_result_for_x" = "No results found for \"%s\"";
-
+"search_no_result_for_x" = "No results found for \"%1$@\"";
```

Comment:
we cannot change this here, the string will arrive as %s again, esp. in translations. 
but i think this can be handled by ./tools/tx-pull-translations.sh

Reply:
The issue is that the android tranlsation (strings.xml) uses a wrong placeholder. It should be `%1$s` instead of `%s` and therefore the regex doesn't recognizes the placeholder in order to replace it by the iOS conform `%1$@`. I will create a pull request in the Android repository for it.

Reply:
I think these kind of edge cases shouldn't be covered by ./tools/tx-pull-translations.sh. I would prefer to have a valid input file as a precondition for the translation process.

Reply:
i disagree here. most input files come from transifex and maybe other sources, we cannot guarantee "clean input" here.

for robustness, we should adapt the regex and accept %s as an equivalent to %1$s - even if this is not 100% correct - it is the format used in printf() and easily done wrong therefore. and, of course, we should not crash.

be strict what you write and accepting what you read :)

Reply:
done